### ISSUE TITLE
Can't load config class with name 'typescript-axios'

### ISSUE THREAD

**Issue by tanakaworld** [OpenAPITools/openapi-generator-cli#11]
In version `0.0.8-3.2.0` I got this error.
How can I use [typescript-axios generator](https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators/typescript-axios.md)?

```
Can't load config class with name 'typescript-axios'
Available:
ada
ada-server
android
apache2
apex
aspnetcore
bash
clojure
cwiki
cpp-qt5
cpp-qt5-qhttpengine-server
cpp-pistache-server
cpp-restbed-server
cpp-restsdk
cpp-tizen
csharp
csharp-dotnet2
csharp-nancyfx
dart
eiffel
elixir
elm
erlang-client
erlang-server
flash
scala-finch
go
go-server
groovy
kotlin
kotlin-server
haskell-http-client
haskell
java
jaxrs-cxf-client
java-inflector
java-msf4j
java-pkmst
java-play-framework
java-undertow-server
java-vertx
jaxrs-cxf
jaxrs-cxf-cdi
jaxrs-jersey
jaxrs-resteasy
jaxrs-resteasy-eap
jaxrs-spec
javascript
javascript-flowtyped
javascript-closure-angular
jmeter
lua
nodejs-server
objc
openapi
openapi-yaml
perl
php
php-laravel
php-lumen
php-slim
php-silex
php-symfony
php-ze-ph
powershell
python
python-flask
r
ruby
ruby-on-rails
ruby-sinatra
rust
rust-server
scalatra
scala-akka
scala-httpclient
scala-gatling
scala-lagom-server
scalaz
spring
dynamic-html
html
html2
swift2-deprecated
swift3
swift4
typescript-angular
typescript-angularjs
typescript-aurelia
typescript-fetch
typescript-inversify
typescript-jquery
typescript-node

[error] Check the spelling of the generator's name and try again.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

**Comment by tanakaworld** [comment]
This error seems to be fixed in version `0.0.8-4.0.0-beta2` 🤔

**Comment by kay-schecker** [comment]
It's seems not to be an issue. I'd say the generator you'd like to use was just not available in v3.2.0 of https://github.com/OpenAPITools/openapi-generator

**Comment by wing328** [comment]
Right, v3.2.0 was pretty old so please update to the latest version to get all the generators.

**Comment by sumanthakannantha** [comment]
> Right, v3.2.0 was pretty old so please update to the latest version to get all the generators.

Facing same issue with https://oss.sonatype.org/content/repositories/snapshots/org/openapitools/openapi-generator-cli/6.0.0-SNAPSHOT/

> It's seems not to be an issue. I'd say the generator you'd like to use was just not available in v3.2.0 of https://github.com/OpenAPITools/openapi-generator

Facing same issue with the latest version
https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/5.2.1/
